release_notes/41-release-notes.adoc

```diff
@@ -371,6 +371,12 @@ to:
 
 Network graph 1.0 has been removed after the depreciation notice in {product-title-short} 4.0. Network graph 2.0 performs the same functionality and is in GA.
 
+[id="notice-upcoming-report-change_{context}"]
+== Notice of upcoming change for reports
+
+In version 4.0, {product-title-short} released the collections feature that replaced access scopes used in report configurations. {product-title-short} automatically created equivalent collections for access scopes used in existing report configurations and migrated report configurations to use newly-created collections. If the migration fails, the report configurations become non-functional, and {product-title-short} logs the error messages in Central logs and in the {product-title-short} web portal.
```

Comment:
For the third sentence, would past tense  "If ACS failed to create an equivalent collection for an access scope, associated report configurations became non-functional, and ACS logged the error messages in Central logs..." be better?

Reply:
Yes, or change the second sentence to present tense also, but I think past tense for all works well.